src/gui/pages_menu/KM_GUIMenuMultiplayer.pas

```diff
@@ -560,9 +566,14 @@ procedure TKMMenuMultiplayer.MP_ServersDoubleClick(Sender: TObject);
 
 
 procedure TKMMenuMultiplayer.MP_HostClick(Sender: TObject);
+var
+  serverPortStr: string;
+  serverPort: Word;
 begin
   //Save the player and IP name so it is not lost if something fails
   MP_SaveSettings;
+  serverPortStr := IcsTrim(Edit_MP_ServerPort.Text);
+  serverPort    := atoi(serverPortStr);
```

Comment:
Both 'IcsTrim' and 'atoi' are introduced by the class 'OverbyteIcsUtils'.
It is also used by Ics, hence I used it as well.
I decided to follow their method to assure having the same result.

I think using StrToInt would be just fine.

Useful link : [atoi in OverbyteIcsUtils](https://github.com/Kromster80/kam_remake/blob/master/src/common/Overbyte%20ICS%208/OverbyteIcsUtils.pas#L2511)